Makefile

```diff
@@ -7,11 +7,11 @@ version=$(shell cat VERSION)
 
 .PHONY: livepeer
 livepeer:
-	go build -ldflags="-X github.com/livepeer/go-livepeer/core.LivepeerVersion=$(version)-$(shell git describe --always --long --dirty)" cmd/livepeer/*.go
+	go build -ldflags="-X github.com/livepeer/go-livepeer/core.LivepeerVersion=$(version)-$(shell git describe --always --long --dirty --abbrev=8)" cmd/livepeer/*.go
```

Comment:
rather than:
```
git describe --always --long --dirty --abbrev=8
```
may I suggest:
```
git rev-parse --short HEAD
```

Reply:
@mkrufky but it doesn't show `dirty` status, if I'm right